app/Controllers/Graph.hs

```diff
@@ -50,7 +50,7 @@ getGraphJSON = do
     response <- liftIO $ getGraph graphName
     withDefault response
     where
-        withDefault (Just response) = return response
+        withDefault (Just response) = return $ createJSONResponse response
```

Comment:
Now that `return $ createJSONResponse` are common to both branches, you have an opportunity to delete this function altogether and replace Line 51 with a single line.

Hint: use https://hackage.haskell.org/package/base-4.20.0.1/docs/Data-Maybe.html#v:maybe

Reply:
I submitted this change. However, the line is really long. I thought I could instead define a local variable instead of having it in a single line. I am unsure on what is the best practice here.
`let defaultValue = createJSONResponse $ object ["texts" .= ([] :: [Text]),
                    "shapes" .= ([] :: [Text]),
                   "paths" .= ([] :: [Text])]`